draft-ietf-quic-transport.md

```diff
@@ -2152,35 +2155,40 @@ receive packets without first having sent a packet on that path. Effective NAT
 traversal needs additional synchronization mechanisms that are not provided
 here.
 
-An endpoint MAY include PATH_CHALLENGE and PATH_RESPONSE frames that are used
-for path validation with other frames.  In particular, an endpoint can pad a
-packet carrying a PATH_CHALLENGE for Path Maximum Transfer Unit (PMTU)
-discovery (see {{pmtud}}), or an endpoint can include a PATH_RESPONSE with its
-own PATH_CHALLENGE.
+An endpoint MAY include other frames with the PATH_CHALLENGE and PATH_RESPONSE
+frames used for path validation.  In particular, an endpoint can include
+PADDING with a PATH_CHALLENGE for Path Maximum Transfer Unit (PMTU) discovery
+(see {{pmtud}}), and/or include a PATH_RESPONSE with its own PATH_CHALLENGE.
```

Comment:
This last bit sounded backwards to me in the original text. I think as a server I expect to commonly receive an unprompted PATH_CHALLENGE, and will therefore have to send PATH_RESPONSE. The performance optimization is to include a PATH_CHALLENGE with that PATH_RESPONSE, not the other way around.

```suggestion
(see {{pmtud}}), and/or include a PATH_CHALLENGE with its own PATH_RESPONSE.
```